README.adoc

```diff
@@ -131,20 +127,47 @@ InventoryLivenessCheck.java
 include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryLivenessCheck.java[]
 ----
 
+A health Check for liveness allows third party services to determine if the application is running. 
+This means that if this procedure fails the application can be discarded (terminated, shutdown).
+The `@Liveness` annotation must be applied on a HealthCheck implementation to define a Liveness 
+check procedure, otherwise, this annotation is ignored.
 This liveness check verifies that the heap memory usage is below 90% of the maximum memory.
 If more than 90% of the maximum memory is used, a status of `DOWN` will be returned. 
 
+The `inventory` microservice should be healthy only when `system` is available. To add this 
+check to the `/health/ready` endpoint, you will create a class that is annotated with the
+`@Readiness` annotation and implements the `HealthCheck` interface. 
+A Health Check for readiness allows third party services to know if the application is ready to process requests or not.
+The `@Readiness` annotation must be applied on a HealthCheck implementation to define a readiness check procedure, otherwise, this annotation is ignored.
+
+[role="code_command hotspot file=2", subs="quotes"] 
+---- 
+#Create the `InventoryReadinessCheck` class.#
+`inventory/src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java`
+---- 
+
+InventoryReadinessCheck.java
+[source, Java, linenums, role='code_column hide_tags=copyright']
+----
+include::finish/inventory/src/main/java/io/openliberty/guides/inventory/InventoryReadinessCheck.java[]
+----
+
+This health check verifies that the `system` microservice is available at 
+`\http://system-service:9080/`. The `system-service` host name is only accessible from 
```

Comment:
The `system-service` host name is only accessible from inside the cluster, you can't access it yourself.
--->
The `system-service` host name is accessible only from inside the cluster; you can't access it yourself.